app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/ConfirmPledgeDetailsScreen.kt

```diff
@@ -741,3 +806,33 @@ fun ItemizedRewardListContainer(
         }
     }
 }
+
+@Composable
```

Comment:
Methods that have a return should not be composable.  We can probably either inline this or make it a util function that takes in a context to get the string.

Reply:
could perfectly be a extracted to `RewardViewUtils`